sdk/eventhub/Azure.Messaging.EventHubs/src/Processor/ProcessEventArgs.cs

```diff
@@ -38,6 +38,12 @@ public struct ProcessEventArgs
         ///   The received event to be processed.  Expected to be <c>null</c> if the receive call has timed out.
         /// </summary>
         ///
+        /// <remarks>
```

Comment:
@ellismg: This is an attempt to clarify the semantics that you had raised as potentially confusing.   Does this make sense, or do you have suggestions for tuning the verbiage to better help?

Reply:
I'm one of the world's worst prose writers, but I wonder if using a term like "immutable" vs "non-volatile" is better?  However - I think this is a fine remark and would have let me know it's safe to cache this.

Reply:
I like "immutable".   I agree that its probably a much more widely understood concept.